src/carnot/funcs/builtins/collections.h

```diff
@@ -82,26 +82,27 @@ class AnyUDA : public udf::UDA {
   void Update(FunctionContext*, TArg val) {
     // TODO(zasgar): We should find a way to short-circuit the agg since we only care
     // about one value.
-    if (!picked) {
-      val_ = val;
-      picked = true;
-    }
+    SetValue(val);
   }
 
-  void Merge(FunctionContext*, const AnyUDA&) {
-    // Does nothing.
-  }
+  void Merge(FunctionContext*, const AnyUDA& other) { SetValue(other.val_); }
```

Comment:
I think you want `if (other.picked) SetValue...` to avoid setting `picked = true` in the case that both `this` and `other` haven't picked a value yet.

Reply:
Fixed in `5fb0b1e`.